ruler-frontend/src/main/kotlin/com/spotify/ruler/frontend/components/Ownership.kt

```diff
@@ -74,8 +72,33 @@ fun RBuilder.componentOwnershipOverview(components: List<AppComponent>) {
 }
 
 @RFunction
-fun RBuilder.componentOwnershipPerTeam(components: List<AppComponent>, sizeType: Measurable.SizeType) {
```

Comment:
Maybe we can also avoid having two different paths for `xxx` and `xxxWithoutFiles` here, I think it should be possible to simplify this a little bit.

Reply:
sure, let me give it another try

Reply:
Now there is only on code-path, let me know what you think